core/tests/protractor_utils/ExplorationPlayerPage.js

```diff
@@ -128,7 +128,7 @@ var ExplorationPlayerPage = function() {
   // This verifies the question just asked, including formatting and
   // rich-text components. To do so the richTextInstructions function will be
   // sent a handler (as given in forms.RichTextChecker) to which calls such as
-  //   handler.readItalicText('slanted');
+  // handler.readItalicText('slanted');
```

Comment:
I think the indentation was intentional, to offset this line of code from the prose surrounding it.

Reply:
Done, thanks!